ui/src/app/services/context.service.ts

```diff
@@ -68,6 +70,11 @@ export class ContextService {
         this.allUsers = userList;
     }
 
+    setAllManagers(userList) {
+        // let managers = _.filter(userList, 'manager');
```

Comment:
Can this line go away or be implemented?

Reply:
removing comment.